src/Domain/CreateInitialElement.hpp

```diff
@@ -14,11 +14,15 @@ template <size_t Dim>
 class ElementId;
 /// \endcond
 
+namespace domain {
+namespace Initialization {
 /*!
- * \ingroup ComputationalDomainGroup
+ * \ingroup InitializationGroup
```

Comment:
As long as you are touching this file can you address #1556 by adding the extra documentation I suggest?

Reply:
Let’s keep PRs small. Adding that documentation is nontrivial. I pushed a suggestion to #1619.